WcaOnRails/config/routes.rb

```diff
@@ -58,7 +58,13 @@
   get 'polls/:id/vote' => 'votes#vote', as: 'polls_vote'
   get 'polls/:id/results' => 'polls#results', as: 'polls_results'
 
-  resources :teams, only: [:index, :new, :create, :update, :edit]
+  resources :committees do
+    resources :teams, only: [:new, :edit, :create, :update, :destroy] do
+      resources :team_members, only: [:new, :edit, :create, :update]
+    end
+    resources :committee_positions, only: [:index, :new, :edit, :create, :update, :destroy]
```

Comment:
This messes up my simple_form.

`simple_form_for [@committee, @position]`

This tries to find committee_committee_path but if I pass in the URL option, it is not the same for the edit and the new. Is there a way to deal with this?